### PR TITLE
Update Manage POM Cases Service

### PR DESCRIPTION
**Pull request by rj-adams**
Update with revised component definitions

### CODE REVIEW

File: src/main/kotlin/model/ManagePOMCases.kt [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          Tags.DATABASE.addTo(this)
+          CloudPlatform.rds.add(this)
+      }
+
+      managePOMCasesAPI = system.addContainer(
+        "manage POM Cases API",
+        "API exposing data mastered by the MPC Service",
+        "Ruby On Rails"
+      ).apply {
+        setUrl("https://moic.service.justice.gov.uk/api-docs/index.html")
```
Comment (sldblog) [inline src/main/kotlin/model/ManagePOMCases.kt:65]: By convention at this time, these URLs point to the GitHub repo of the application -- could it point there too, please? We can make this better/more explicit later

**Review by sldblog — COMMENTED** [review]
Looks nice 👍

Could the commit/PR title be updated with the intent of the change, please? (Updating the prison-probation case handover?)

File: src/main/kotlin/model/ManagePOMCases.kt [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        "MPC database",
+        "Database holding data mastered by MPC",
+        "AWS RDS PostgreSQL"
+      ).apply {
+          Tags.DATABASE.addTo(this)
+          CloudPlatform.rds.add(this)
+      }
+
+      managePOMCasesAPI = system.addContainer(
+        "manage POM Cases API",
```
Comment (sldblog) [inline src/main/kotlin/model/ManagePOMCases.kt:61]: 💭 Is this the same as `allocationManager`? The container (deployable) could serve the purpose of a web app, REST API and other functions, we don't have to separate them.

**Review by sldblog — APPROVED** [review]
👍

File: src/main/kotlin/model/ManagePOMCases.kt (hunk 2 above)
Comment (rj-adams) [reply src/main/kotlin/model/ManagePOMCases.kt:61]: The API is exposed from the same "block" of code which exposes the UI.

The service is written in RoR app which natively encourages a service orientated approach but within a single monolithic application - I'm looking to expose the internal breakdown of this monolith into it's constituent components - help/guidance on this very welcome!!

**Pull request merged**